src/mac/CMakeLists.txt

```diff
@@ -10,11 +10,15 @@ if (NRN_MACOS_BUILD)
   set(MACPKG_INSTALL_PATH "/Applications/NEURON-${PROJECT_VERSION}")
   if (${CMAKE_INSTALL_PREFIX} STREQUAL ${MACPKG_INSTALL_PATH})
     # should only be invoked after a 'make install'. Creates signed pkg.
+    set(PKG_NAME "NEURON-${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}")
+    set(PKG_PATH "${CMAKE_CURRENT_BINARY_DIR}/build/${PKG_NAME}.pkg")
     add_custom_target(macpkg
```

Comment:
Not sure if it would be doable, but could it be possible to split the target into:
* `macpkg`- create `nrn.pkg`
* `notarize`- notarize and staple the `nrn.pkg`

(what I'm trying to follow is the separation between generic actions and those bound to a specific Apple dev ID). I guess not, since the pkg contents must be signed individually and we cannot sign the full pkg at once?